infra/terraform/hash/prod-usea1.tfvars

```diff
@@ -28,7 +28,7 @@ kratos_env_vars = [
 ]
 
 hash_graph_env_vars = [
-  { name = "HASH_GRAPH_ALLOWED_URL_DOMAIN_PATTERN", secret = false, value = "https://alpha.hash.ai/@(?P<shortname>[\\w-]+)/types/(?P<kind>(?:data-type)|(?:property-type)|(?:entity-type))/[\\w-]+/" },
+  { name = "HASH_GRAPH_ALLOWED_URL_DOMAIN_PATTERN", secret = false, value = "https://alpha.hash.ai/@(?P<shortname>[\\w-]+)/types/(?P<kind>(?:data-type)|(?:property-type)|(?:entity-type))/[\\w\\-_%]+/" },
```

Comment:
Some of the keys have weird characters like `+`. I hope by URL encoding them we only need to add support for these characters, we'll find out with time I guess